go/vm/lfvm/instructions.go

```diff
@@ -814,6 +814,12 @@ func opCreate2(c *context) {
 		return
 	}
 
+	// Charge for the code size
+	words := (size.Uint64() + 31) / 32
```

Comment:
if this change is a fix on the VM it should be a separate PR.

Reply:
There was no newly found bug in the VM implementation. But the changes performed in #446 to CREATE where also applied to CREATE2. This was done without testing the CREATE2 changes and the order had to be updated which is done in this PR.